### PR TITLE
Updating clone command to receive directory as argument

### PR DESCRIPTION
**Pull request by lucasncabral**
In this PR the `--folder` option was removed from the clone command. To maintain support for creating directories, the optional argument `[DIRECTORY]` has been added. 

```
Usage: ml-git clone [OPTIONS] REPOSITORY_URL [DIRECTORY]

  Clone an ml-git repository ML_GIT_REPOSITORY_URL

Options:
  --untracked  Does not preserve git repository tracking.
  --verbose    Debug mode
```

### CODE REVIEW

File: ml_git/commands/general.py [hunk 1]
```diff
@@ -23,9 +23,9 @@ def mlgit():
 
 @mlgit.command('clone', help='Clone a ml-git repository ML_GIT_REPOSITORY_URL')
```
Comment (Vieira-Carlos) [inline ml_git/commands/general.py:24]:
```suggestion
@mlgit.command('clone', help='Clone an ml-git repository ML_GIT_REPOSITORY_URL')
```

File: docs/mlgit_commands.md [hunk 2]
```diff
@@ -802,14 +802,13 @@ You should only use this command for the flexible mutability option.
 <br>
 
 ```
-Usage: ml-git clone [OPTIONS] REPOSITORY_URL
+Usage: ml-git clone [OPTIONS] REPOSITORY_URL [DIRECTORY]
 
   Clone a ml-git repository ML_GIT_REPOSITORY_URL
```
Comment (Vieira-Carlos) [inline docs/mlgit_commands.md:807]:
```suggestion
  Clone an ml-git repository ML_GIT_REPOSITORY_URL
```

**Review by MatheusClaudi — APPROVED** [review]
Checked

File: ml_git/commands/general.py (hunk 1 above)
Comment (lucasncabral) [reply ml_git/commands/general.py]: The message was updated at [4dc2928](https://github.com/HPInc/ml-git/pull/232/commits/4dc29285051f198e82aa3bb9e47ef34812ba56fe).

File: docs/mlgit_commands.md (hunk 2 above)
Comment (lucasncabral) [reply docs/mlgit_commands.md]: The message was updated at 4dc2928.

**Review by Vieira-Carlos — APPROVED** [review]

**Review by andersonpablosilva — APPROVED** [review]

**Pull request merged**